ui/public/locales/pt_BR.json

```diff
@@ -1,879 +1,1143 @@
+"label.copy.text": "Copiar texto",
+"label.copyid": "Copiar ID",
 "label.copying.iso": "Copiando ISO",
 "label.corrections.saved": "Altera\u00e7\u00f5es salvas",
 "label.counterid": "Contador",
 "label.cpu": "CPU",
-"label.cpu.allocated": "CPU Alocada",
-"label.cpu.sockets": "Sockets",
-"label.cpuallocated": "CPU Alocada por VMs",
+"label.cpu.allocated": "CPU alocada",
+"label.cpu.sockets": "Sockets da CPU",
+"label.cpuallocated": "CPU alocada por VMs",
 "label.cpuallocatedghz": "Alocado",
 "label.cpulimit": "Limite de CPU",
 "label.cpumaxdeviation": "Desvio",
-"label.cpusockets": "O N\u00famero de Sockets de CPU",
+"label.cpunumber": "N\u00facleos da CPU",
+"label.cpusockets": "O n\u00famero de sockets de CPU",
 "label.cpuspeed": "CPU (em MHz)",
-"label.cputotal": "CPU TOTAL",
+"label.cputotal": "CPU total",
 "label.cputotalghz": "Total",
-"label.cpuused": "CPU Utilizada",
-"label.create.nfs.secondary.staging.storage": "Cria Armazenamento NFS de Est\u00e1gio Secund\u00e1rio",
-"label.create.nfs.secondary.staging.store": "Criar storage staging secund\u00e1rio NFS",
+"label.cpuused": "CPU utilizada",
+"label.cpuusedghz": "CPU utilizada",
+"label.create.account": "Criar conta",
+"label.create.backup": "Iniciar backup",
+"label.create.network": "Criar nova rede",
+"label.create.nfs.secondary.staging.storage": "Cria armazenamento NFS de est\u00e1gio secund\u00e1rio",
 "label.create.project": "Criar um projeto",
+"label.create.project.role": "Criar fun\u00e7\u00e3o para o projeto",
+"label.create.site.vpn.connection": "Criar conex\u00e3o VPN site-para-Site",
+"label.create.site.vpn.gateway": "Criar gateway de VPN site-para-Site",
+"label.create.snapshot.for.volume": "Criar snapshot para volume",
 "label.create.ssh.key.pair": "Criar par de chaves SSH",
 "label.create.template": "Criar template",
+"label.create.tier.aclid.description": "A ACL associada \u00e0 camada",
+"label.create.tier.externalid.description": "ID da rede em um sistema externo",
+"label.create.tier.gateway.description": "Gateway da camada no intervalo superior do CIDR, n\u00e3o sobrepondo o CIDR das outras camadas dessa VPC.",
+"label.create.tier.name.description": "Um nome \u00fanico para a camada",
+"label.create.tier.netmask.description": "M\u00e1scara de rede da camada. Por exemplo 255.255.255.0",
+"label.create.tier.networkofferingid.description": "A oferta de rede da camada",
+"label.create.user": "Criar usu\u00e1rio",
+"label.create.vpc.tier": "Criar camada VPC",
 "label.create.vpn.connection": "Criar uma conex\u00e3o VPN",
 "label.created": "Criado",
 "label.created.by.system": "Criado pelo sistema",
-"label.createnfscache": "Cria Armazenamento NFS de Est\u00e1gio Secund\u00e1rio",
-"label.credit": "Cru00e9dito",
-"label.crosszones": "Inter Zonas",
-"label.current": "isCurrent",
-"label.currentpassword": "Senha Antiga",
+"label.createnfscache": "Cria armazenamento NFS de est\u00e1gio secund\u00e1rio",
```

Comment:
Shouldn't it be "Criar" instead of "Cria"?

Reply:
I could not find the usage for this label, but it makes more sense to be `Criar` for sure.